### ISSUE TITLE
GNU/Linux distributions link is broken

### ISSUE THREAD

**Issue by ivanosevitch** [MicrosoftDocs/WSL#812]
> Choose your favorite GNU/Linux distributions [from the Microsoft Store](https://aka.ms/wslstore)

This link https://aka.ms/wslstore gives an error:
![image](https://user-images.githubusercontent.com/15523557/88453542-3fc4bc80-ce68-11ea-88f7-7e3240bbf9aa.png)

Note:
- I'm not logged in to https://docs.microsoft.com when clicking on the link.
- Other links such as https://aka.ms/wslubuntu2004 work fine

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3be148fb-e4a9-83b5-5b5a-7834eed7052f
* Version Independent ID: f583bc11-fe60-bb2c-af2b-dc0dae9fe81f
* Content: [About Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about)
* Content Source: [WSL/about.md](https://github.com/MicrosoftDocs/wsl/blob/master/WSL/about.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by dimithras** [comment]
Works for me. Seems like you have some problems with access to [MS Store](https://www.microsoft.com/en-us/store/apps/windows)

**Comment by ivanosevitch** [comment]
I think the issue was due to my browsing of this website using macOS. I intended to install WSL on a Windows computer but were first browsing the instructions on a mac.

Indeed, while the error in my first post came from browsing with Firefox on macOS, I tried agin with Chrome on macOS and got this new and more meaningful error when requesting https://aka.ms/wslstore :
`Failed to launch 'ms-windows-store://collection/?CollectionId=LinuxDistros' because the scheme does not have a registered handler.`
![image](https://user-images.githubusercontent.com/15523557/93050411-215a9080-f663-11ea-9e49-43dd4dd794b5.png)

As we can expect, the `ms-windows-store` protocol cannot be understood by macOS. I'm thus closing the issue.
